nautobot/dcim/templates/dcim/location.html

```diff
@@ -6,7 +6,7 @@
 
 {% block extra_breadcrumbs %}
     {% for ancestor in object.ancestors %}
-        <li>{{ ancestor|hyperlinked_object:"name"}}</li>
+        <li class="breadcrumb-item">{{ ancestor|hyperlinked_object:"name"}}</li>
```

Comment:
Does it make sense to not do anything but the generics? We are updating everything to the component UI on develop, this is going to result in many merge conflicts, so trying to weigh out between the different options.